iavl/utils.py

```diff
@@ -330,26 +335,26 @@ def visit_iavl_nodes(
     stack: List[bytes] = [hash]
     while stack:
         hash = stack.pop()
-        if isinstance(hash, tuple):
-            # already expanded, (hash, node)
+        if isinstance(hash, PersistedNode):
+            # the postorder case, it's already expanded as PersistedNode
             yield hash
             continue
 
         node = get_node(hash)
 
         if not preorder:
```

Comment:
not sure if we need postorder now, but need loop all anyway right?

Reply:
postorder is only used in import/export, which we haven't implemented in python version.

Reply:
what loop all?

Reply:
unless prune_check is different

Reply:
yes, without prune_check it'll traverse all.